### ISSUE TITLE
lesson outline numbering

### ISSUE THREAD

**Issue by psteinb** [carpentries-incubator/hpc-intro#114]
I just noticed that the [lesson-outline.md](https://github.com/hpc-carpentry/hpc-intro/blob/0b34055fe82c1927f5334b26773dacf7ed8f91a5/lesson-outline.md) contains some flaws related to numbering as of https://github.com/hpc-carpentry/hpc-intro/commit/0b34055fe82c1927f5334b26773dacf7ed8f91a5. I didn't catch this during review.

```
* [15-Filesystems and Storage]: This topic didn't get fleshed out in the call. But objectives likely include items from PStein's [Shared Filesystem lesson](https://github.com/psteinb/hpc-in-a-day/blob/gh-pages/_episodes/01-04-shared-filesystem.md):
    * Understand the difference between a local and shared/network filesystem
    * Learn about high performance / scratch Filesystems
    * Raise the attention that misuse (intentional or not) of a common file system neatively affects all users very quickly.
    * Possible tools: echo $TEMP, ls -al /tmp, df, quota

* [15-transferring-files.md: Transferring Files](_episodes/15-transferring-files.md)
    * Understand the (cognitive) limitations that remote systems don't necessarily have local Finder/Explorer windows
    * Be mindful of network and speed restrictions (e.g. cannot push from cluster; many files vs one archive)
    * Know what tools can be used for file transfers, and transfer modes (binary vs text)
    * Objective: Be able to transfer files to and from a computing cluster.
    * Tools: wget, scp, rsync (callout), mkdir, FileZilla, 
    * Remove: dos2unix, unix2dos,
    * Bonus: gzip, tar, dos2unix, cat, unix2dos, sftp, pwd, lpwd, put, get, 
    * Later:
```
@devbioinfoguy could you please have a look and provide a fix based on the google doc?

**Comment by devbioinfoguy** [comment]
So this was a bit of a problem. According to the GDoc, we have a FileSystems/Storage cell, and then a Transferring Files cell. The latter had lesson objectives that were taken from existing content. But the Filesystems/Storage module does not exist; and the objectives are blank. I seem to recall that we felt this module was necessary, and at the time we skipped over filling it out in the interest of time, and would go back later to fill in. 

When editing the outline doc, I filled in objectives that I thought could be a starting point based on past Intro HPC sessions that I've run. But there was not an appropriate way to handle the numbering. I thought I would leave that for the team to resolve on what is appropriate numbering, and why this numbering anyhow.

Does this help, @psteinb?? Happy to chat further about this.

-B
